hatchery-bootstrap-aorg-arm64/Dockerfile

```
@@ -1,8 +1,7 @@
 FROM --platform=linux/arm64 continuumio/miniconda3
 
-RUN apt update
-RUN apt install -y curl git make
+RUN apt update && apt install -y cmake
 RUN . /opt/conda/bin/activate && \
     conda create -c numba/label/dev -c ad-testing/label/py314 -c defaults \
-    -y -n buildenv llvmdev=15 python=3.14 numpy \
+    -y -n buildenv llvmdev=20 python=3.14 numpy \
     gcc_linux-aarch64 gxx_linux-aarch64 setuptools
```

user needs to manually activate `buildenv` on exec after build is complete, otherwise base uses 3.13.
```
+ exec bash
(base) root@7a952bdc9e01:/# python --version
Python 3.13.2
(base) root@7a952bdc9e01:/# which python
/opt/conda/bin/python
(base) root@7a952bdc9e01:/# conda activate buildenv
(buildenv) root@7a952bdc9e01:/# which python 
/opt/conda/envs/buildenv/bin/python
(buildenv) root@7a952bdc9e01:/# python --version
Python 3.14.0rc2
```
can we add env activation on bashrc to not have user remember to activate `buildenv` everytime ?
```suggestion
    gcc_linux-aarch64 gxx_linux-aarch64 setuptools

    RUN echo 'source /opt/conda/bin/activate' >> /root/.bashrc
    RUN echo 'conda activate buildenv' >> /root/.bashrc
```

---

on the containers that derive from the `continuumio/miniconda3` image, we only need to activate the `buildenv` because the miniconda3 distribution is automatically "activated".

---

```
 💣 zsh» docker run -it hatchery-bootstrap-aorg-arm64
(buildenv) root@3d061fef3c75:/#
```

---

implemented in 023937894d3fb2be6de15698cc46b8a5dc78b749